sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -175,6 +175,12 @@ async def create_container(
             has changed, and act according to the condition specified by the `match_condition` parameter.
         :keyword match_condition: The match condition to use upon the etag.
         :paramtype match_condition: ~azure.core.MatchConditions
+        :keyword List[Dict[str, str]] computed_properties: Computed properties must be at the top level in the item and
+            can't have a nested path. Each computed property definition has two components: a name and a query.
+            The name is the computed property name, and the query defines logic to calculate the property value
+            for each item. Computed properties are scoped to an individual item and therefore can't use values
+            from multiple items or rely on other computed properties. Every container can have
+            a maximum of 20 computed properties.
```

Comment:
didn't see this before but this is way too much information for the kwarg - look at the Java doc for this functionality for instance:
<img width="777" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/30335873/ce0ff963-33c4-430b-8bea-26ded11aa2cd">

Reply:
Thank you for catching this. I have updated them to match the Java docstrings more.